vtex-apps/store-components
# InfoCard improvements

#### What is the purpose of this pull request?

### Added
- InfoCard new property `blockClass`.

 ### Fixed
- InfoCard schema.
- InfoCard CMS labels.


#### What problem is this solving?

1. Make it possible to edit InfoCard using the CMS
2. Fix CMS Labels
3. Make it possible to customize each info card instance with CSS using `blockClass`

#### How should this be manually tested?

You can see the InfoCard with two different `blockClass`s here:
https://breno2--storecomponents.myvtex.com/

You can edit the InfoCard in the CMS here:
https://breno2--storecomponents.myvtex.com/admin/cms/storefront

#### Screenshots or example usage

![image](https://user-images.githubusercontent.com/284515/55817011-f103df80-5ac9-11e9-8d3e-e8ad29faef0b.png)

![image](https://user-images.githubusercontent.com/284515/55817034-feb96500-5ac9-11e9-9898-10bbf3c4ded7.png)


#### Types of changes
